tests/acceptance/test_replay_detail.py

```diff
@@ -83,3 +83,57 @@ def test_simple(self):
             self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
             self.browser.snapshot("replay detail")
+
+    def test_dom_events_tab(self):
+        with self.feature(FEATURE_NAME):
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
+            self.browser.click('[data-test-id="DOM Events"]')
+            self.browser.wait_until_test_id("replay-details-dom-events-tab")
+            self.browser.snapshot("replay details - dom events tab")
+
+    def test_console_tab(self):
+        with self.feature(FEATURE_NAME):
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
+            self.browser.click('[data-test-id="Console"]')
+            self.browser.wait_until_test_id("replay-details-console-tab")
+            self.browser.snapshot("replay details - console tab")
+
+    def test_network_tab(self):
+        with self.feature(FEATURE_NAME):
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
+            self.browser.click('[data-test-id="Network"]')
+            self.browser.wait_until_test_id("replay-details-network-tab")
+            self.browser.snapshot("replay details - network tab")
+
+    def test_memory_tab(self):
+        with self.feature(FEATURE_NAME):
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
+            self.browser.click('[data-test-id="Memory"]')
+            self.browser.wait_until_test_id("replay-details-memory-tab")
+            self.browser.snapshot("replay details - memory tab")
+
+    def test_errors_tab(self):
+        with self.feature(FEATURE_NAME):
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
+            self.browser.click('[data-test-id="Errors"]')
+            self.browser.wait_until_test_id("replay-details-errors-tab")
+            self.browser.snapshot("replay details - errors tab")
+
+    def test_trace_tab(self):
+        with self.feature(FEATURE_NAME):
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
+            self.browser.click('[data-test-id="Trace"]')
+            self.browser.wait_until_test_id("feature-disabled-message")
```

Comment:
If you want to see the trace tab itself, instead of the disabled message, then use `with self.feature(['organizations:performance-view']);` or concat that 2nd feature with the original list.

It could matter more if there were multiple tabs that could render with 'feature-disabled-message' and we wanted to distinguish between them, but in this case it's fine either way imo.

I found that feature name by looking at the wrapper for the trace view here: https://github.com/getsentry/sentry/blob/8803d531e39ba14d4d2b643ea5faea0e30351bdc/static/app/views/replays/detail/trace/index.tsx#L13